### PR TITLE
Marketplace/add fbs delivery note attr to order

### PR DESCRIPTION
**Pull request by alexandrosth**
Documentation for `fbs_delivery_note` attribute for fulfilled by skroutz orders when dispatched from warehouse

### CODE REVIEW

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 1]
```diff
@@ -21,6 +21,7 @@ Name         | Type   | Value | Description
 `accept_options` | Object | | [Accept options](#accept-options) (available for orders with state `"open"`)
 `reject_options` | Object | | [Reject options](#reject-options) (available for orders with state `"open"`)
 `fulfilled_by_skroutz` | Boolean | | Whether the order is fulfilled by skroutz
+`fbs_delivery_note` | String | | Order's delivery note from skroutz warehouse
```
Comment (tassosMaurides) [inline source/localizable/smart_cart/_order_object.html.md.erb:24]: `for fulfilled by Skroutz orders`

Also maybe Skroutz with capital S

Comment (nikostsaganos) [reply source/localizable/smart_cart/_order_object.html.md.erb:24]: `for "fulfilled by Skroutz" orders` 
or `orders that are fulfilled by Skroutz`

Comment (alexandrosth) [reply source/localizable/smart_cart/_order_object.html.md.erb]: yes you're right 
768a8b12229c36a23a158649c94470b0928ceea1
897abc1fbb2af64db994bdfb87453682aeb0ff0e

Comment (alexandrosth) [reply source/localizable/smart_cart/_order_object.html.md.erb]: Updated 
f365907cfd4b358ffd95746fcaac23e75d54c356

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #### Order object
 
-Name         | Type   | Value | Description
------------- | ------ | ----- | -----------------
+Name         | Type | Value | Description
+------------ | -- | ----- | -----------------
```
Comment (tassosMaurides) [inline source/localizable/smart_cart/_order_object.html.md.erb:4]: Is this related to this feature?

**Review by tassosMaurides — APPROVED** [review]
:+1:

File: source/localizable/smart_cart/_order_object.html.md.erb (hunk 2 above)
Comment (alexandrosth) [reply source/localizable/smart_cart/_order_object.html.md.erb]: no missed (ide automations ..) fixed aefe4c1731b3be55cf0f41436ab6260857fb4057
